### PR TITLE
Add Emergency Shotgun Cabinet

### PR DESCRIPTION
**Pull request by ps3moira**
## About the PR

I added a shotgun cabinet.
## Why / Balance

I always thought it'd be nice for sec outposts to have something to make it feel more complete, as well as something for sec officer's incase they run into danger but are too far from the Sec Department to run over there.
## Media

Screenshot 2023-11-03 145331

Screenshot 2023-11-03 145730

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: Added emergency shotgun cabinet

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml [hunk 1]
```diff
@@ -166,7 +166,9 @@
     size: Normal
   - type: BallisticAmmoProvider
     capacity: 4
-
+  - type: Tag
+  id: WeaponShotgunKammerer
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml:170]:
```suggestion
    id: WeaponShotgunKammerer
```

**Comment by CatMagic1** [comment]
epic as shit

**Comment by ps3moira** [comment]
> epic as shit
> 
> does it start loaded with 2 lethal shells?

It's start filled with 4 lethal rounds.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml [hunk 2]
```diff
@@ -166,7 +166,9 @@
     size: Normal
   - type: BallisticAmmoProvider
     capacity: 4
-
+  - type: Tag
+      id: WeaponShotgunKammerer
```
Comment (FluffiestFloof) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml:170]:
```suggestion
  - type: Tag
    tags:
      - WeaponShotgunKammerer
```

**Comment by UbaserB** [comment]
Looks great

**Comment by IProduceWidgets** [comment]
I question the wisdom of easily steal-able shotguns when shotguns are pretty spectacular compared to a pistol or disabler (or really even a rifle).

I like the sprite though!

**Comment by CatMagic1** [comment]
> I question the wisdom of easily steal-able shotguns when shotguns are pretty spectacular compared to a pistol or disabler (or really even a rifle).
> 
> I like the sprite though!
> 
> Also accepting a double barrel would be neat too. Not sure how hard that would be to pull off.

You would pretty much just have to add WeaponShotgunDoubleBarreled under both whitelists. Would probably need separate sprites for that though.

**Comment by brainfood1183** [comment]
cabinet should be id locked if it isnt already

**Comment by CatMagic1** [comment]
> cabinet should be id locked if it isnt already

locked with security and command access

**Comment by ps3moira** [comment]
I did a mess-up, the same for another PR where I accidentally merged submodule and other stuff that wasn't supposed to, so since this is pretty easy PR I will just remake it.

**Comment by Emisse** [comment]
you can revert the submodule commit

File: Resources/Prototypes/Entities/Structures/Wallmounts/shotgun_cabinet.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    closedState: glass
+
+- type: entity
+  id: ShotGunCabinetFilled
+  parent: ShotGunCabinet
+  suffix: Filled
+  components:
+  - type: ItemCabinet
+    cabinetSlot:
+      startingItem: WeaponShotgunKammerer
```
Comment (CaptainSqrBeard) [inline Resources/Prototypes/Entities/Structures/Wallmounts/shotgun_cabinet.yml:63]: Security outposts could be looted by powergamers. It probably should be at least double-barreled shotgun, yet it's still be powergame-able. This should not be an issue with these cabinets in brig (i.e. warden office).

**Pull request closed without merging**